src/NuGet.Protocol.Catalog/SimpleHttpClient.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class SimpleHttpClient : ISimpleHttpClient
```

Comment:
nit: why not `JsonHttpClient` or similar to reflect the JSON deserialization that the class does?

Reply:
My hope is that other convenience methods could be added here as well. For example, fetching the .nuspec as `XDocument` or a .nupkg as a local temp file. `Json` is just the first convenience needed.